spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/instrument/web/client/feign/TraceFeignAspect.java

```diff
@@ -53,6 +53,8 @@ public Object feignClientWasCalled(final ProceedingJoinPoint pjp) throws Throwab
 			log.debug("Executing feign client via TraceFeignAspect");
 		}
 		if (bean != wrappedBean) {
+			// NOTE: in master(3813cf9dd47f98db0e075412abbffbd9d6742974),
+			// this is executeTraceFeignClient(wrappedBean, pjp)
 			return executeTraceFeignClient(bean, pjp);
```

Comment:
This should be `wrappedBean` I guess

Reply:
yep just it blows up tests if we fix it:)

Reply:
Can't we align those tests as we did in master?

Reply:
master swapped out ribbon so it changes the nature of the tests I think. that's why I paused here

Reply:
Ok I see. Let's merge it as it is then and I'll look at this once I have spare time. It won't make things worse obviously ;)